src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/service/CellCertificateService.kt

```diff
@@ -58,7 +58,7 @@ class CellCertificateService(
   fun getCellCertificate(id: UUID): CellCertificateDto {
     val cellCertificate = cellCertificateRepository.findById(id)
       .orElseThrow { CellCertificateNotFoundException(id) }
-    return CellCertificateDto.from(cellCertificate)
+    return CellCertificateDto.from(cellCertificate, showLocations = true)
```

Comment:
Only return all the location data when looking at an individual certificate or the current one